include/library/spdm_lib_config.h

```diff
@@ -87,4 +87,13 @@
 #define LIBSPDM_ENABLE_CAPABILITY_KEY_EX_CAP 1
 #define LIBSPDM_ENABLE_CAPABILITY_PSK_EX_CAP 1
 
+/* Worst case round-trip transport timing in microseconds */
+#define LIBSPDM_RTT_VALUE               10000
```

Comment:
Please remove it and expose set_data() API.

Reply:
For both RTT and CT I don't think having them as `#define`s provides much optimization. I think it's fine to remove them from `spdm_lib_config.h` and just use `set_data`.